consensus/core/src/transaction.rs

```diff
@@ -229,6 +234,10 @@ impl TransactionVerifier for NoopTransactionVerifier {
     fn verify_batch(&self, _batch: &[&[u8]]) -> Result<(), ValidationError> {
         Ok(())
     }
+
+    fn vote_batch(&self, _batch: &[&[u8]]) -> Vec<TransactionIndex> {
+        vec![]
+    }
```

Comment:
Do we need to return a here a rejection reason as well? I know that as part of the protocol we don't look to include the rejection reasons in the DAG (although it would be convenient now that we won't have quorum driver gathering the responses) , but it might be useful? I guess we can deal with that if needed later.